src/UseFlash.tsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
```

Comment:
This is just an example of how to use the render props, right? 

I think we should remove this component, and instead put the example in one of the stories. We could do that in a follow up PR though, it doesn't need to be part of this PR.